components/logo/MainLogo.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useUser, useSubscription, useOrganization, Href } from 'react-components';
+
+import { formatPlans } from '../../containers/payments/subscription/helpers';
+
+const MainLogo = ({ product, url = 'https://mail.protonmail.com/' }) => {
+    const [{ isFree, hasPaidMail, hasPaidVpn } = {}, loadingUser] = useUser();
+    const [{ Theme } = {}, loadingOrganization] = useOrganization();
+    const [{ Plans } = {}, loadingSubcription] = useSubscription();
+
+    const { mailPlan = {}, vpnPlan = {} } = formatPlans(Plans);
+
+    if (Theme) {
+        return (
+            <Href url={url} rel="noreferrer help" className="logo-container nodecoration flex-item-centered-vert">
+                <img alt="custom-logo" src={Theme} />
```

Comment:
You need to parse the `Theme` CSS variable to extract the URL I think.

Reply:
we are leaving aside this for the moment, since it's not clear how to store the CSS